package_ctx.go

```diff
@@ -39,9 +39,13 @@ import (
 //     var (
 //         pctx = NewPackageContext("path/to/blah")
 //
+//         // Should appear in the generated ninja file as "g.path.to.blah.myPrivateVar = abcdef"
```

Comment:
Maybe not worth mentioning here, but live_tracker.go strips the variables and rules down to ones that are referenced through a build statement.